code/modules/antagonists/traitor/objectives/final_objective/supermatter_cascade.dm

```diff
@@ -20,13 +20,13 @@
 	replace_in_name("%AREA%", initial(dest_crystal_area_pickup.name))
 	return TRUE
 
-/datum/traitor_objective/final/end_reality/generate_ui_buttons(mob/user)
+/datum/traitor_objective/final/supermatter_cascade/generate_ui_buttons(mob/user)
 	var/list/buttons = list()
 	if(!sent_crystal)
-		buttons += add_ui_button("", "Pressing this will call down a pod with the universe ending kit.", "biohazard", "destabilizing_crystal")
+		buttons += add_ui_button("", "Pressing this will call down a pod with the supermatter cascade kit.", "biohazard", "destabilizing_crystal")
 	return buttons
 
-/datum/traitor_objective/final/end_reality/ui_perform_action(mob/living/user, action)
+/datum/traitor_objective/final/supermatter_cascade/ui_perform_action(mob/living/user, action)
 	. = ..()
 	switch(action)
 		if("destabilizing_crystal")
```

Comment:
just a little below here, there's some text that calls the destabilizing crystal a "bioterror agent" that's probably from some copy+pasted code. if you could fix that, that'd be swell.